Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -368,7 +382,7 @@ import BraintreeDataCollector
             hostAndPath.append("/")
         }
         
```

Comment:
Now we have some return URL parsing happening in `BTPayPalClient` and some happening in `BTPayPalAppSwitchReturnURL`. Maybe for another PR, we can repurpose `BTPayPalAppSwitchReturnURL` to do URL parsing for _both_ the web flow & app switch flows.

Reply:
Created DTBTSDK-3751 for this